templates/traefik.yaml

```diff
@@ -39,6 +39,12 @@ spec:
         enabled: true
         enforced: true
         generateTLS: true
+        # dex service is exposed with TLS certificate signed by self signed root
+        # Dex CA certificate. It is not clear if traefik supports configuring
+        # trusted certificates per backend. This should be investiaged in a
+        # separate issue.
+        # See: https://jira.mesosphere.com/browse/DCOS-56033
+        insecureSkipVerify: true
```

Comment:
the default is `insecureSkipVerify: false` that will affect other usages of traefik :/. 
I don't like that but let's move forward and address it in a different PR. Perhaps we have to set this value for traefik when Dex is enabled.